libchart/values.yaml

```diff
@@ -14,7 +14,49 @@ image:
   # pullSecrets:
   #   - secret1
   #   - secret2
-
+sidecars:
+  - name: helper1
+    image:
+      repository: "repo"
+      tag: "123"
+      extraContainerPorts: [8088, 9099]
+    securityContext:
+      readOnlyRootFilesystem: true
+      allowPrivilegeEscalation: false
+    resources:
+      limits:
+        cpu: 100m
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+    environment: { }
+    secrets: { }
+    volumeMounts: [ ]
+    # - name: secret-volume
+    #   mountPath: /app/secrets
+    #   readOnly: true
+  - name: helper2
+    image:
+      repository: "repo"
+      tag: "456"
+      extraContainerPorts: [8088, 9099]
+    securityContext:
+      readOnlyRootFilesystem: true
+      allowPrivilegeEscalation: false
+    resources:
+      limits:
+        cpu: 100m
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+    environment: { }
+    secrets: { }
+    volumeMounts:
+      - name: secret-volume
+        mountPath: /app/secrets
+        readOnly: true
```

Comment:
This will force everyone using this chart to use this settings. If they do not explicit override or give a null value they will get this values. There is other company's using this helm chart so this is no go. use sidecars: {} as value and comment out the "example" (see csi: {})

Reply:
Same here.